examples/pxScene2d/src/shell.js

```diff
@@ -20,24 +20,32 @@ var isDuk=(typeof Duktape != "undefined")?true:false;
 var isV8 = (typeof _isV8 != "undefined")?true:false;
 
 px.import({ scene: 'px:scene.1.js',
-             keys: 'px:tools.keys.js',
+  log: "px:tools.../Logger.js",
+  keys: 'px:tools.keys.js',
   url: 'url',
   querystring: 'querystring'
 }).then( function ready(imports)
 {
   var scene = imports.scene;
   var keys  = imports.keys;
-  var queryStringModule = imports.querystring;
   var urlModule = imports.url;
+  var queryStringModule = imports.querystring;
+  var rtl = imports.log.rtl;
+  var Logger = imports.log.Logger;
+  var setLoggingLevel = imports.log.setLoggingLevel;
+
+  var logger = new Logger('shell.js');
+  var rtLogLevel = process.env.RT_LOG_LEVEL ?  process.env.RT_LOG_LEVEL : 'warn';
+  setLoggingLevel(rtl(rtLogLevel));
 
   function uncaughtException(err) {
     if (!isDuk && !isV8) {
-      console.log("Received uncaught exception " + err.stack);
+      logger.message(rtl('error'),"Received uncaught exception " + err.stack);
```

Comment:
Please see the comment above.  Let's remove the developer's need to call `rtl()` here.  The new usage would look like: `logger.message('error',"Received uncaught exception " + err.stack);`.  Please make this change for all other logger.message() updates in this PR as well (that use rtl).